### DISCUSSION TITLE
Getting the seed value from saved data

### DISCUSSION THREAD

**Discussion by alt168** [voxeltycoon/issues#311] [General] [answered]
It would be nice if there was a way to check the seed value from the saved data when I want to start from scratch in the same region.

**Comment by andrewpey** [comment, marked as answer]
You can load a saved game, then head to Game settings, and copy a seed from there. Is not that what are you looking for?

**Comment by alt168** [comment]
There it is! Sorry, I didn't know I can check the game settings during the game. Thank you!
